core/arch/asm_defines.asm

```diff
@@ -376,6 +374,7 @@ ASSUME fs:_DATA @N@\
 # define REG_R10 x10
 # define REG_R11 x11
 # define REG_R12 x12
+# define SYSNUM_REG w8
```

Comment:
This seems to override the definition for `defined(MACOS) && defined(AARCH64)` at L133. Perhaps move that here as well?

Reply:
I move it here, too.